go.mod

```diff
@@ -29,36 +29,36 @@ require (
 	k8s.io/klog/v2 v2.30.0
 	k8s.io/kube-openapi v0.0.0-20211115234752-e816edb12b65
 	k8s.io/kubectl v0.0.0
-	k8s.io/kubernetes v0.0.0
+	k8s.io/kubernetes v1.23.4
```

Comment:
Not sure why this is now showing a real version? This is after `go mod tidy` and even if I put in `v0.0.0` manually, it keeps coming back as `v1.23.4`. @sttts any ideas?

Reply:
Still not sure what's up here...

Reply:
That is weird, but I also don't know that it matter since it gets `replace`d below anyway.